templates/taxi/manufacturer_confirm_delete.html

```diff
@@ -0,0 +1,9 @@
+{% extends "base.html" %}
+{% block content %}
+  <h1>Delete manufacturer</h1>
+  <h3>Are you sure want to delete manufacturer: {{ manufacturer.name }}</h3>
+  <form action = "" method="post">
```

Comment:
The `action` attribute of the form is empty. You should set this attribute to the correct URL using Django's `{% url %}` template tag to ensure proper form submission. For example: `{% url 'manufacturer-delete' manufacturer.id %}`.